src/codebase.md

```diff
@@ -260,3 +260,41 @@ crates optimally for your semantic needs.
 | [ring](https://docs.rs/ring/)           | the leading crypto library         |
 | [nalgebra](https://docs.rs/nalgebra/)   | linear algebra                     |
 | [once_cell](https://docs.rs/once_cell/) | complex static data                |
+
+## How should I call C++ functions from Rust and vice-versa?
+
+Behind the scenes, Rust natively supports calling C functions using `extern "C"`,
+`#[repr(C)]` and `#[no_mangle]`. This is, of course, error prone and involves
+lots of `unsafe` code. So don't do that: there are better ways.
+
+Your initial decision should be a matter of policy. Choose between:
+
+* I want to manually specify the cross-language interface in order to keep
```

Comment:
I find the switch from "You" in the previous paragraph to "I" in this one a bit jarring. Maybe just use "You" in these bullet points too?

Reply:
Done